pyproject.toml

```diff
@@ -1,14 +1,14 @@
 [tool.poetry]
 name = "db-client"
-version = "3.9.7"
+version = "3.11.11"
```

Comment:
I'd prefer not to have done this as part of this PR, but we need to to get some of the nice validation, and it needs to be done.

My bad, and I'll deal with any fallout when updating dependencies.